### ISSUE TITLE
Can we *re-name* this repo?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/CRM#3]
CRM is a very much ["Businessese"](https://en.wiktionary.org/wiki/businessese) initialisation that _few_ people outside of "sales & marketing" know about.
I'm thinking of re-naming the repo from "CRM" to something a bit more "lay-person friendly".

I initially thought of "**people**", because describes what it's ***for***.

We could call it "**contact**" because a CRM system is all about establishing/maintaining contact with "contacts" ... but then why not just stick with "people"? 🤔

It could _also_ be named "**com**" (_as in **com**munity, or **com**munications_)
because that's what the app will ***be***.
it's also _shorter_ and is more likely to become a "verb":
https://www.fastcompany.com/3004901/google-what-it-means-when-brand-becomes-verb
e.g: "_did you **com** that?_" (_did you enter those details into the **com** app?_)

Another option could be "***NetWork***" because the _real_ aim of a CRM (_despite what Salesforce wants it's customers to believe_) is to create a NetWork of people who can help each other achieve their goals. Relevant: https://github.com/dwyl/phase-two/blob/master/NetWork.md

@iteles thoughts?

**Comment by iteles** [comment]
Agreed, I don't think the _name_ is the most important thing to spend time on right now.
